Helper/Entity/Product/PriceManager/ProductWithChildren.php

```diff
@@ -158,28 +158,50 @@ protected function setFinalGroupPrices($field, $currencyCode, $min, $max, $dashe
     {
         if (count($subproducts) > 0) {
             $groupPriceList = [];
+            $subProductsMin = -1;
```

Comment:
This seems a bit unclear. Can we maybe use named constants here like `PRICE_NOT_SET`? Is there a reason that both variables are initialized differently?